<!doctype html>
<html>
  <head>
    <title>LGSF Dashboard</title>
    <link rel="stylesheet"  href="/lgsf-dashboard/assets/css/styles.css">
    <link rel="shortcut icon" href="https://democracyclub.org.uk/static/images/logo_icon.png">
  </head>
  <body>
    <div class="ds-page">
      <a class="ds-skip-link" href="#main">skip to content</a>

      <header class="ds-header">
        <a class="ds-skip-link" href="#main">skip to content</a>
        <a class="ds-logo" href="/lgsf-dashboard/">
          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
          <span>LGSF Logbooks</span>
        </a>

      </header>


      <main id="main" tabindex="-1" class="ds-stack">

        
<style>

    pre {
        height:400px;
        overflow-x: scroll;
        width:100%
    }
</style>




  


  <h2 id="2023-10-13-10-35">2023-10-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-13 10:35:03.268150</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-13 10:35:05.644618</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[10:35:03] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[10:35:04] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[10:35:05] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-13-09-40">2023-10-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-13 09:40:36.460253</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-13 09:40:40.095052</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[09:40:36] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[09:40:38] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[09:40:39] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[09:40:40] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-12-12-44">2023-10-12</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-12 12:44:04.927317</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-12 12:44:08.080543</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:44:04] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:44:05] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:44:06] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:44:07] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[12:44:08] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-11-12-30">2023-10-11</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-11 12:30:58.858846</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-11 12:31:01.106471</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:30:58] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:30:59] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:31:00] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[12:31:01] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-10-13-43">2023-10-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-10 13:43:32.327338</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-10 13:43:34.636815</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:43:32] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:43:33] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:43:34] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-09-12-04">2023-10-09</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-09 12:04:57.744906</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-09 12:04:59.820717</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:04:57] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:04:58] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:04:59] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-08-14-11">2023-10-08</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>1 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-08 14:11:39.990916</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-08 14:11:41.986967</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:11:39] Fetching Scraper for: HUN                              handlers.py:23
[14:11:40] Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:11:41] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-07-12-36">2023-10-07</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>1 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-07 12:36:18.748446</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-07 12:36:20.743742</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:36:18] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:36:19] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:36:20] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-06-12-51">2023-10-06</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-06 12:51:12.115281</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-06 12:51:14.143941</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:51:12] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:51:13] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:51:14] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-05-12-13">2023-10-05</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-05 12:13:40.527144</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-05 12:13:42.636738</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:13:40] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[12:13:41] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:13:42] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-04-13-57">2023-10-04</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-04 13:57:42.930536</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-04 13:57:45.087213</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:57:42] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[13:57:43] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[13:57:44] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[13:57:45] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-03-13-09">2023-10-03</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-03 13:09:16.301481</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-03 13:09:19.845012</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:09:16] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[13:09:18] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:09:19] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-02-12-37">2023-10-02</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-02 12:37:33.236772</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-02 12:37:35.372039</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:37:33] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:37:34] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:37:35] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-01-12-56">2023-10-01</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-01 12:56:52.981061</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-01 12:56:55.017134</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:56:52] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:56:53] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:56:54] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[12:56:55] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-30-13-51">2023-09-30</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-30 13:51:27.911828</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-30 13:51:30.137345</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:51:27] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[13:51:28] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:51:29] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[13:51:30] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-29-13-14">2023-09-29</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-29 13:14:54.643912</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-29 13:14:56.836796</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:14:54] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:14:55] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:14:56] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-28-12-33">2023-09-28</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-28 12:33:14.117653</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-28 12:33:16.445906</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:33:14] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:33:15] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:33:16] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-27-14-28">2023-09-27</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-27 14:28:25.008172</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-27 14:28:27.408934</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:28:25] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[14:28:26] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:28:27] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-26-13-00">2023-09-26</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-26 13:00:34.666720</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-26 13:00:36.682187</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:00:34] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:00:35] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:00:36] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-25-12-14">2023-09-25</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-25 12:14:34.351283</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-25 12:14:36.627916</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:14:34] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[12:14:35] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:14:36] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  


      </main>
      <footer class="ds-footer">
        <div class="ds-block-centered ds-text-centered ds-stack">
          <div class="ds-cluster-center">
            <ul>
              <li><a href="/lgsf-dashboard/api/failing.json">Failing JSON</a></li>
            </ul>
          </div>
          <div class="ds-copyright">
            <a href="https://democracyclub.org.uk/">
              <img src="https://DemocracyClub.github.io/design-system/images/logo_white_text.svg" alt="Democracy Club Home" />
            </a>
            <p>Copyright © 2021 Democracy Club</p>
            <p>Community Interest Company</p>
            <p>Company No: <a href="https://beta.companieshouse.gov.uk/company/09461226">09461226</a></p>
          </div>
        </div>
      </footer>
    </div>
  </body>
